test/extended/quota/clusterquota.go

```diff
@@ -189,6 +199,74 @@ var _ = g.Describe("[sig-api-machinery][Feature:ClusterResourceQuota]", func() {
 
 				t.Fatalf("unexpected error: %v", err)
 			}
+
+			// test templates are counted correctly
+			template := &templatev1.Template{}
```

Comment:
this test confirms the already merged https://github.com/openshift/cluster-policy-controller/pull/100